============================= test session starts ==============================
platform darwin -- Python 3.12.0, pytest-7.4.4, pluggy-1.4.0
rootdir: /private/var/folders/z0/zvd81kyj08xgnt8c1hbb2f780000gn/T/pytest-of-whenessel/pytest-3/test_pytest_marker_with_kwargs_utf_encoding0
plugins: allure-pytest-2.13.4.dev2+gd759bc5
collected 1 item

test_pytest_marker_with_kwargs_utf_encoding.py .                         [100%]

=============================== warnings summary ===============================
../../../../../../../../../Users/whenessel/Development/PycharmProjects/allure-python/.venv/lib/python3.12/site-packages/_pytest/config/__init__.py:735
  /Users/whenessel/Development/PycharmProjects/allure-python/.venv/lib/python3.12/site-packages/_pytest/config/__init__.py:735: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: allure_pytest
    self.import_plugin(arg, consider_entry_points=True)

test_pytest_marker_with_kwargs_utf_encoding.py:3
  /private/var/folders/z0/zvd81kyj08xgnt8c1hbb2f780000gn/T/pytest-of-whenessel/pytest-3/test_pytest_marker_with_kwargs_utf_encoding0/test_pytest_marker_with_kwargs_utf_encoding.py:3: PytestUnknownMarkWarning: Unknown pytest.mark.marker - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
    @pytest.mark.marker(stuff='—è')

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
======================== 1 passed, 2 warnings in 0.00s =========================
